innersource-activity-score.md

```diff
@@ -0,0 +1,92 @@
+## Title 
+InnerSource Activity Score
+
+## Patlet
+The InnerSource activity score is a numeric value that represents the (GitHub) activity of an InnerSource project. It can be used to create a ranked list of projects for the [InnerSource portal](innersource-portal.md) so that potential contributors can easily find the most active projects. It is derived automatically from GitHub repository statistics and may be enriched with KPIs from other tools or manual evaluations. 
+
+## Problem
+
+Potential contributors want to find active projects with a lot of traction, but also fairly new and enthusiastic projects with a need for new contributions - easy and fast.
+
+**But in which order** shall InnerSource projects be presented? Typical ranking KPIs like *GitHub Stars*, *Number of Forks*, *Number of Commits*, *Lines of Code*,  *Last Update* aren't very accurate when it comes to activity of a project.
+
+A new metric derived from several KPIs is needed to define a reliable and versatile score for a project's activity level.
+It can be used to sort or rank projects according to their activity level.
```

Comment:
For brevity I would just use one of the words order/sort/rank to describe the main feature that the activity score enables.

```suggestion
It can be used to sort projects according to their activity level.
```